src/aks-platform/03_aks_0.tf

```diff
@@ -24,7 +24,7 @@ module "snet_aks_platform" {
 
 
 module "aks" {
-  source = "git::https://github.com/pagopa/terraform-azurerm-v3.git//kubernetes_cluster?ref=kuernetes-improve-outputs"
+  source = "git::https://github.com/pagopa/terraform-azurerm-v3.git//kubernetes_cluster?ref=kubernetes_cluster_module_test"
```

Comment:
Can we use a tag version instead of a branch name?

Reply:
sure but the problem here is we are using the parameter `ignore_changes` for `microsoft_defender["log_analytics_workspace_id"]` in this branch and I don't know what I have to do with it